### DISCUSSION TITLE
Can i add glow effect to lineChart in recharts

### DISCUSSION THREAD

**Discussion by nguyenquan265** [recharts/recharts#5376] [Q&A]
``Hello everybody,
I just started using this library and I like it a lot.
But I have one problem in this using.
I want to create a linechart like this:
![image](https://github.com/user-attachments/assets/356eb771-f26c-4869-b777-59365802756d)

This is my working result is below:
![image](https://github.com/user-attachments/assets/0efd34c0-2b34-4371-869a-5d78a54b358d)

My code is:
`import {
  Line,
  LineChart,
  XAxis,
  YAxis,
  CartesianGrid,
  ReferenceLine,
  ResponsiveContainer
} from 'recharts'

const data = [
  { time: '17:06:30', price: 3475.0 },
  { time: '17:06:40', price: 3473.5 },
  { time: '17:06:50', price: 3476.2 },
  { time: '17:07:00', price: 3478.0 },
  { time: '17:07:10', price: 3476.8 },
  { time: '17:07:20', price: 3476.42 },
  { time: '17:07:30', price: 3476.0 },
  { time: '17:07:40', price: 3475.83 },
  { time: '17:07:50' },
  { time: '17:08:00' }
]

const CustomizedDot = (props: any) => {
  const { cx, cy, value } = props

  if (value == 3476.8) {
    return (
      <svg>
        <circle cx={cx} cy={cy} r={5} fill='green' />
        <circle cx={cx} cy={cy} r={3} />
      </svg>
    )
  }

  if (value == 3475.83) {
    return (
      <svg>
        <defs>
          <filter id='pink-shadow' x='-50%' y='-50%' width='200%' height='200%'>
            <feGaussianBlur in='SourceAlpha' stdDeviation='5' />
            <feDropShadow
              dx='0'
              dy='0'
              stdDeviation='0.1'
              floodColor='#dd7978'
            />
          </filter>
        </defs>

        <circle cx={cx} cy={cy} r={7} fill='red' filter='url(#pink-shadow)' />

        <circle cx={cx} cy={cy} r={6} fill='#ff6761' />
      </svg>
    )
  }

  return null
}

const Home = () => {
  return (
    <div className='w-full h-screen bg-[#131828] p-6'>
      <ResponsiveContainer width='100%' height='100%'>
        <LineChart
          data={data}
          margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
        >
          <CartesianGrid vertical={true} stroke='#202734' />

          <XAxis
            dataKey='time'
            axisLine={false}
            tickCount={8}
            tickSize={0}
            tickLine={true}
            tick={{ fill: '#8d909b' }}
            tickMargin={10}
            padding={{ right: 50 }}
          />

          <YAxis
            domain={['auto', 'auto']}
            orientation='right'
            axisLine={false}
            tickFormatter={(value) => value.toFixed(2)}
            tickCount={7}
            tickSize={0}
            tickLine={false}
            tick={{ fill: '#8d909b' }}
            tickMargin={16}
          />

          <ReferenceLine
            x={'17:07:10'}
            stroke='#bca4a8'
            strokeDasharray='3 3'
            label={{ value: 'Current', fill: 'hsl(0 0% 100%)' }}
          />

          <ReferenceLine y={3476.8} stroke='#bca4a8' strokeDasharray='3 3' />

          <Line
            type='monotone'
            dataKey='price'
            stroke='#d7f172'
            strokeWidth={1}
            activeDot={false}
            dot={<CustomizedDot />}
          />
        </LineChart>
      </ResponsiveContainer>
    </div>
  )
}

export default Home
`

Please give me a good answer.
Thank you

**Comment by ckifer** [comment]
Something like this should work

https://github.com/recharts/recharts/issues/4026#issuecomment-1865187735 (just do that for line instead of area)


```css
.my-line {
  filter: drop-shadow(1px 1px 5px #8884d8);
}
```

```tsx
<Line {...props} className="my-line" />
```
